_layouts/eip.html

```diff
@@ -41,4 +41,31 @@ <h1 class="page-heading">
 
   {{ content }}
 
+  <h2>Citation</h2>
+  <p>Please cite this document as:</p>
+  {% comment %}
+  IEEE specification for reference formatting:
+  https://ieee-dataport.org/sites/default/files/analysis/27/IEEE%20Citation%20Guidelines.pdf
+  {% endcomment %}
+  <p>{% include authorlist.html authors=page.author %}, "EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status != "Final" %} [DRAFT]{% endif %}," <em>Ethereum Improvement Proposals</em>, no. {{ page.eip | xml_escape }}, {{ page.created | date: "%B %Y" }}. [Online serial]. Available: https://eips.ethereum.org/EIPS/eip-{{ page.eip | xml_escape }}.</p>
 </div>
+
+{% comment %}
+Article schema specification:
+https://schema.org/Article
+{% endcomment %}
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "author": "{% include authorlist.html authors=page.author %}",
+    "name": "EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status != "Final" %} [DRAFT]{% endif %}",
+    "dateCreated": "{{ page.created | date: "%Y-%m-%d" }}",
+{% if page["discussions-to"] != undefined %}
+    "discussionUrl": "{{ page["discussions-to"] | uri_escape }}",
+{% endif %}    
+    "inLanguage": "en-US",
+    "license": "https://creativecommons.org/publicdomain/zero/1.0/",
```

Comment:
While it is the recommended one for EIPs, I think this should be a copy of the license section of each EIP. Is it possible to omit this field to facilitate a quick merge?

Reply:
> While it is the recommended one for EIPs

Not just recommended, *required*.  Currently, all EIPs **MUST** be CC0 (which is why I indicated in that other thread that merging a GPL EIP would require winning holy war to get EIP-1 changed).

Reply:
@axic please advise if this fixes the issue or if I should remove it for faster merge.

We can always merge then add stuff later, not a big deal.

Reply:
Ok, it says "must be in public domain", so CC0 or any other statement of public domain is valid. However, that line was added to EIP-1 **after** a bunch of EIPs were merged lacking a license statement, see #2062.

I think conceptually this PR is doing two things, so would prefer to deal with it separately, especially as there might be more discussion points around the schema.

Reply:
To be clear this is what the **current** EIP-1 states:
```
Copyright Waiver - All EIPs must be in the public domain. See the bottom of this EIP for an example copyright waiver.
```

It does not say it must be CC0, so using CC0 in the license line in my opinion is not 100% correct.

Reply:
Great point. Due to an underspecification on schema.org, I have found that linking to the other part of the page is legal. Maybe it shouldn't be. That's their fault.

Fixed in 0f70b6b0fed62c8c350899b13235cdabc74463d3

Is this agreeable?